### PR TITLE
UG - Include labelled images for set functions 

### PR DESCRIPTION
**Pull request by PraveenElango**
Include labelled images for set functions in UG for both circuit and boolean templates.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -229,16 +231,7 @@ Example of usage:
...
-The resistor was set to 500.0 Ω
-	+---R-----C---+
-	|             |
-	|             |
-	+----+V_ac+---+
-Current Voltage: 0.0 V
-Total Resistance: 500.0 Ω
-Total Capacitance: 0.0 µF
-```
+![SetCircuit](diagrams/SetCircuitLabel.png)
```
Comment (sevenseasofbri) [inline docs/UserGuide.md:231]: For this image could you change the green arrow to point at "the resistor was set to 500" instead of total resistance:
image

Comment (PraveenElango) [reply docs/UserGuide.md:231]: Okay, changed!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -229,16 +231,7 @@ Example of usage:
 `set r 500`
 
 Expected Outcome:
```
Comment (sevenseasofbri) [inline docs/UserGuide.md]: Perhaps instead of having example of usage + expected output, could you combine it into 1 sentence that reads something like: "the following diagram is an example of when `set r 500` command is used in a valid template"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -339,15 +332,8 @@ Example of usage:
 
 Expected outcome:
```
Comment (sevenseasofbri) [inline docs/UserGuide.md:330]: Same for this

File: docs/UserGuide.md (hunk 2 above)
Comment (PraveenElango) [reply docs/UserGuide.md]: Okay, changed.

File: docs/UserGuide.md (hunk 3 above)
Comment (PraveenElango) [reply docs/UserGuide.md:330]: Okay, changed.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -163,7 +165,7 @@ reached the end of the tutorial.
 
 When you have typed in a command that is inconsistent with what is required as per the tutorial's
 instruction for that particular step, an invalid command message will be returned until you type 
-in the correct command in the terminal line as shown below. 
+in the correct command in the terminal line as shown below. r
```
Comment (hughjazzman) [inline docs/UserGuide.md:168]: ??
```suggestion
in the correct command in the terminal line as shown below.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -271,7 +261,7 @@ This command adds a component, in a specific configuration - either *in parallel
 
 The image below shows an example of using the command `add parallel r 500` on a template.
 
-![AddCircuit](diagrams/AddCircuitLabel.png)
+![AddCircuit](diagrams/SetCircuitLabel.png)
```
Comment (hughjazzman) [inline docs/UserGuide.md]: Why is this changed?
```suggestion
![AddCircuit](diagrams/AddCircuitLabel.png)
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -138,6 +138,8 @@ The image below shows the expected summary table shown with the command `summary
 
 ![Summary](diagrams/Summary.png)
 
+
+
```
Comment (hughjazzman) [inline docs/UserGuide.md:142]:
```suggestion
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -224,21 +226,9 @@ This command sets the value of a component. The component must be part of the cu
     * `v` represents a voltage source
 * The `VALUE` can be an integer of a float, of which the units depends on the component set.
 
-Example of usage:
-
-`set r 500`
+The following diagram is an example of when the `set r 500` command is used in any circuit template:
```
Comment (hughjazzman) [inline docs/UserGuide.md]: For consistency.
```suggestion
The image below shows an example of using the command `set r 500` on the template shown.
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -367,20 +350,9 @@ This command sets the value of an input. The inputs will then be used in the `ca
...
-Example of usage:
-
-`set B 0`
-
-Expected Outcome:
-```
-      OUT
-       |
-   B       C
+The following diagram is an example of when the `set B 0` command is used in any boolean template:
```
Comment (hughjazzman) [inline docs/UserGuide.md]:
```suggestion
The image below shows an example of using the command `set B 0` on the template shown.
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -367,20 +350,9 @@ This command sets the value of an input. The inputs will then be used in the `ca
...
 
-OUT = B AND C
-B = 0
-C = ?
-```
+![AddBoolean](diagrams/SetBooleanLabel.png)
```
Comment (hughjazzman) [inline docs/UserGuide.md:355]:
```suggestion
![SetBoolean](diagrams/SetBooleanLabel.png)
```

**Review by hughjazzman — CHANGES_REQUESTED** [review]
